pkgs/development/python-modules/furl/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, pythonPackages }:
```

don't import pythonPackages, rather the individual python packages `flake8, six, orderedmultidict`
```suggestion
{ stdenv, buildPythonPackage, fetchPypi, flake8, six, orderedmultidict }:
```